llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -531,7 +531,6 @@ class VPSingleDefRecipe : public VPRecipeBase, public VPValue {
     case VPRecipeBase::VPWidenIntOrFpInductionSC:
     case VPRecipeBase::VPWidenPointerInductionSC:
     case VPRecipeBase::VPReductionPHISC:
-    case VPRecipeBase::VPScalarCastSC:
```

Comment:
VPInstructionSC above now also holds for VPInstructionWithType recipes, instead of VPScalarCastSC.